salt/modules/pkgin.py

```diff
@@ -295,6 +313,28 @@ def list_pkgs(versions_as_list=False, **kwargs):
     return ret
 
 
+def list_upgrades(refresh=True, **kwargs):
+    '''
+    List all available package upgrades.
+
```

Comment:
This could use a `.. versionadded:: Oxygen` tag here for this new function.

Reply:
And done